Resources/config/sitemap.xml

```diff
@@ -7,12 +7,6 @@
 
     <parameters>
         <parameter key="cmf_seo.sitemap.controller.class">Symfony\Cmf\Bundle\SeoBundle\Controller\SitemapController</parameter>
-        <parameter key="cmf_seo.sitemap.provider.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Provider</parameter>
-
-        <parameter key="cmf_seo.sitemap.loader_chain.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Loader</parameter>
-        <parameter key="cmf_seo.sitemap.guesser.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Guesser</parameter>
-        <parameter key="cmf_seo.sitemap.voter_chain.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\Voter</parameter>
-        <parameter key="cmf_seo.sitemap.phpcr.public_workflow_voter.class">Symfony\Cmf\Bundle\SeoBundle\Sitemap\PublishWorkflowUrlInformationVoter</parameter>
```

Comment:
Isn't that just a "Fabians best practice"? And on the other hand the best practices suggest that behavior for Applications only and allow it for third party bundles.